.github/workflows/os-osd-deployment.yml

```diff
@@ -0,0 +1,131 @@
+# This workflow was used for deploying the latest version of OpenSearch and OpenSearch Dashboards to AWS EKS (Elastic Kubernetes Service) cluster.
+name: Deploy OpenSearch and OpenSearch Dashboards 
```

Comment:
this this action for prod or dev?

Reply:
this action is for prod and dev both, which depended on dev or prod or both configuration change

Reply:
Thanks